### ISSUE TITLE
Any plans to support desktop scrolling too?

### ISSUE THREAD

**Issue by vgoklani** [roeierez/infinite-list#6]
I would like to have one codebase that runs on both :)

**Comment by roeierez** [comment]
Yes, I got several requests for desktop support, currently I am working on supporting auto detect height of list items so I will probably work on that right after.

**Comment by roeierez** [comment]
The desktop support is now implemented in 'native_scroller' branch.
You can test it by adding a configuration option to the list which is called 'useNativeScroller'.
e.g: 

`list = new InfiniteList({

```
    itemHeightGetter: function(){ return 50;},

    itemRenderer: function(index, domElement){
        domElement.innerHTML = 'Item ' + index;
    },

    initialPage: {
        itemsCount: 200
    }

})`
```

Soon it is going to be merged into master.

**Comment by guo-yu** [comment]
👍

**Comment by ggriffithsIDBS** [comment]
Came across this repo, and this code is working wonderfully for my usecase, however, in the need for the desktop scrolling, any plans to release it?
